assigner/backends/decorators.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from assigner.backends import GitlabBackend, MockBackend
+from assigner.config import requires_config
+
+
+logger = logging.getLogger(__name__)
+
+
+def requires_config_and_backend(func):
```

Comment:
I don't like having this and `requires_config` in different places, but I'm not sure what the right place is for them.

Reply:
Ehh this is a perfectly fine color for this yak shed.